client/packages/invoices/src/Returns/InboundDetailView/columns.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from '@openmsupply-client/common';
 import { InboundReturnLineFragment } from '../api';
 import { InboundReturnItem } from '../../types';
-import { PackVariantCell } from 'packages/system';
+import { getPackVariantCell } from '@openmsupply-client/system';
```

Comment:
Missed renames I guess... don't know why this wasn't picked up the first time and ran fine 👀